.circleci/config.yml

```diff
@@ -55,7 +55,9 @@ jobs:
     steps:
       - checkout
 
-      # Manage rsync
+      # Manage rsync:
+      #   SSH_KNOWN_HOSTS is populated with the output of
+      #   ssh-keyscan -f $REMOTEHOST | base64.
```

Comment:
Se non erro alla fine abbiamo usato:
`ssh-keyscan -h designers.italia.it | base64`

@alexrj confermi?

Reply:
Senza argomenti... correggo ;) `-h` su linux non c'è. Fate sapere se voi con macos ne avete messi.